### ISSUE TITLE
Library request in cdnjs

### ISSUE THREAD

**Issue by extend1994** [alexkuz/react-json-tree#64]
Hi,
There is a [library request](https://github.com/cdnjs/cdnjs/issues/9343) in cdnjs.
To contain the library correctly, I would like to check the files need to added.

The files I plan to add is:
all files in `/src` ( including /src/utils and /src/themes )

Are they complete for the user? Or other files required? Or some files not required?

Thank you for your great work!

**Comment by alexkuz** [comment]
I'm not sure how exactly it should work, I'm not that familiar with cdnjs. Shouldn't it be a prebuilt bundle?
Otherwise - yes, `/src` contains everything required.

**Comment by extend1994** [comment]
cdnjs contains files running on browsers.
So whether files in format like `JSON___Node.js` in `/src` may be for nodejs only but not the browsers?

Thank you for your previous reponse :)

**Comment by alexkuz** [comment]
> cdnjs contains files running on browsers.

Yeah, I get that, but I'm not sure how it will should be built to work with external React etc.
All files are required, it just supposed to be built with a bundler like webpack or browserify.
I think you can use a config to build it like this one:

https://github.com/christianalfoni/formsy-react/blob/master/webpack.production.config.js

**Comment by PeterDaveHello** [comment]
> All files are required, it just supposed to be built with a bundler like webpack or browserify.

Agree, sometimes we just got library hosting request from users don't really know how the libs work, I'm not writing react apps, but I also wonder if this host could help, or is this the proper way to host it.

@alexkuz if you are also not sure, and not going to provide pre-bundled dist files, I think maybe we should just leave it alone, until we know about how to handle it, what do you think?

**Comment by alexkuz** [comment]
True. It's probably not hard to build that dist (and there are probably other react libs in `cdnjs` already, just like that `formsy-react`), but I don't have much time to dig in it right now.

**Comment by PeterDaveHello** [comment]
Maybe we could just leave this issue open for a while, and close it if we can't make it after months, thanks @alexkuz

**Comment by alexkuz** [comment]
Yep, that's fine for me
